api/nakadi-event-bus-api.yaml

```diff
@@ -1634,6 +1671,32 @@ definitions:
       purposes. We recommend logging this object in order to solve connection issues. Clients
       should not parse this structure.
 
+  CursorDistanceQuery:
+    required:
+      - partition
+      - initial_offset
+      - final_offset
+    properties:
+      initial_offset:
+        type: string
+        description: |
+          Offset of the older event in the stream.
+      final_offset:
+        type: string
+        description: |
+          Offset of the newer event in the stream.
+
+  CursorDistanceResult:
+    required:
+      - partition
+      - distance
+    properties:
+      distance:
+        type: number
+        description: |
+          Number of events between two offsets. Initial offset is exclusive. It's only zero when both provided offsets
+          are equal.
```

Comment:
What happens if the older offset is in fact newer than the newer offset? Do we reply with an error, or do we give a negative value for distance?